### PR TITLE
Postsrsd: additional configuration

### PR DESCRIPTION
**Pull request by gkleen**
###### Motivation for this change

postsrsd supports more configuration options than exposed through it's service module (especially `excludeDomains` is needed in my setup)
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/mail/postsrsd.nix [hunk 1]
```diff
@@ -20,17 +20,29 @@ in {
...
-        type = types.path;
-        default = "/var/lib/postsrsd/postsrsd.secret";
-        description = "Secret keys used for signing and verification";
+      separator = mkOption {
+        type = types.enum ["-" "=" "+"];
+        default = "=";
+        description = "First separator character in generated addresses";
       };
 
+      # bindAddress = mkOption {
```
Comment (Mic92) [inline nixos/modules/services/mail/postsrsd.nix]: what's about this option here?

**Comment by pngwjpgh** [comment]
@Mic92, any more comments?
Seeing this included would make maintenance of my machines somewhat easier :wink:

**Comment by Mic92** [comment]
@pngwjpgh sorry I did not noticed your answer.

**Comment by pngwjpgh** [comment]
@Mic92 *bump*?

File: nixos/modules/services/mail/postsrsd.nix [hunk 2]
```diff
@@ -20,15 +20,27 @@ in {
...
         type = types.str;
         description = "Domain name for rewrite";
       };
 
-      secretsFile = mkOption {
-        type = types.path;
-        default = "/var/lib/postsrsd/postsrsd.secret";
-        description = "Secret keys used for signing and verification";
+      separator = mkOption {
+        type = types.enum ["-", "=", "+"];
```
Comment (Mic92) [inline nixos/modules/services/mail/postsrsd.nix]: there should be no comma here.

**Comment by Mic92** [comment]
Thanks!

**Pull request closed without merging**